internal/hcl/funcs/yaml.go

```diff
@@ -0,0 +1,44 @@
+package funcs
+
+import (
+	"strings"
+
+	yaml "github.com/zclconf/go-cty-yaml"
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/function"
+)
+
+var YAMLDecodeFunc = function.New(&function.Spec{
```

Comment:
Might be good to add some docs to these functions to state why we're overriding them.

Reply:
good shout